.gitignore

```diff
@@ -6,3 +6,6 @@ build
 coverage
 
 builderror.log
+
+*.iml
```

Comment:
Can you please remove these changes to `.gitignore`? Ideally these should go in your `.gitignore_global`.